pay-api/src/pay_api/services/fas/routing_slip.py

```diff
@@ -233,3 +233,19 @@ def create(cls, request_json: Dict[str, any], **kwargs):
 
         routing_slip.commit()
         return cls.find_by_number(rs_number)
+
+    @classmethod
+    def update(cls, rs_number: str, action: str, request_json: Dict[str, any]) -> Dict[str, any]:
+        """Update routing slip."""
+        if (patch_action := PatchActions.from_value(action)) is None:
+            raise BusinessException(Error.PATCH_INVALID_ACTION)
+
+        routing_slip: RoutingSlipModel = RoutingSlipModel.find_by_number(rs_number)
+        if routing_slip is None:
+            raise BusinessException(Error.FAS_INVALID_ROUTING_SLIP_NUMBER)
+
+        if patch_action == PatchActions.UPDATE_STATUS:
```

Comment:
hm... so how will the updates work?
will we have more patchactions?

Reply:
i was thinking if we should just update based on the field?

Reply:
That was added to avoid the complexity we are seeing in auth-api right now. 
So that a block of value can be sent, and based on action selected values gets updated. 
https://restful-api-design.readthedocs.io/en/latest/methods.html#patch-vs-put the standards also suggest to go with a formal approach. 
And if we had similar thing in auth-api things would have been much simpler I think.
And we have only update status atm, but things can change quickly as we saw in all of our apps.